src/router.js

```diff
@@ -17,13 +18,33 @@ const guardTipComments = (to, from, next) => {
   }
 };
 
+const guardCommentView = (to, from, next) => {
+  if (to.name === 'comment'
+      && typeof to.params.id !== 'undefined'
+      && typeof to.params.tipId !== 'undefined') {
+    next();
+  } else {
+    next('/');
+  }
+};
+
 const routes = [
   {
     path: '/',
     name: 'home',
     component: TipsList,
     meta: { title: 'Home' },
   },
+  {
+    path: '/tip/:id/comment/:id',
```

Comment:
```suggestion
    path: '/tip/:tipId/comment/:id',
```

Reply:
can we request an API entrypoint for comments? this does not seem logical if comments has unique ID, you don't need the tipId to request the entity data

Reply:
I think we should keep the routes like this, for easier filtering later on in the comment section and also easier associating on-chain and off-chain comments with the specific tip.

Reply:
Also I think this should be done with `route children`

Reply:
https://github.com/aeternity/tipping-community-backend/blob/superhero/routes/README.md